### DISCUSSION TITLE
Flow vs Bag

### DISCUSSION THREAD

**Discussion by MikeKry** [OrchardCMS/OrchardCore#7509] [Q&A]
Hi, I am searching google for a while, but still can't determine difference between flow and bag (other than flow can contain only widgets).

Is bag somehow unsuitable for widgets? Why Flow is not by default reusable (so it could be named and attached multiple times)?

Thanks

**Comment by Piedone** [comment]
The basic difference is that Bag Part is for when you want to put other content items into a content item, emphasis being on hierarchy and storage. Flow Part is about displaying other content items (widgets) within a content item in a structured way (a "flowing" responsive layout), emphasis being on display.
